### PR TITLE
fix code block formatting

### PR DESCRIPTION
**Pull request by greggman**
fixes the code block formatting site wide

See https://github.com/rust-lang/www.rust-lang.org/issues/926

### CODE REVIEW

File: src/styles/app.scss [hunk 1]
```diff
@@ -52,6 +52,19 @@ code {
...
+  line-height: 1.2;
+}
+pre>code {
+  line-height: 1.2;
+  padding: 0;
+  white-space: pre;
+  border: none !important;
+  background: transparent !important;
+  margin: 0;
+}
```
Comment (geophree) [inline src/styles/app.scss]: I think a simpler change would get the same effect.  When I added the following rule to the page in Chrome the issue went away:
```css
pre > code {
  display: block;
  white-space: pre;
}
```

**Comment by Manishearth** [comment]
Should be deployed at http://www-staging.rust-lang.org/ in a bit, please check it out!

**Comment by LordJuss** [comment]
Oh, yes, much better in staging!  Looking forward to seeing it in the actual web-site.

**Comment by erlend-sh** [comment]
Also noticed this on the Networking page and confirmed fixed on staging site.

**Pull request merged**